doc/developer-guide/deployment/https.rst

```diff
@@ -3,9 +3,10 @@
 HTTPS support
 =============
 
-Zotonic has built-in support for https and certificate handling.
+Zotonic has built-in support for HTTPS and TLS (previously SSL) certificate
+handling.
 
-All sites share a single ssl port using Server Name Indication for 
+All sites share a single TLS port using Server Name Indication for
```

Comment:
Zotonic is an HTTP framework, so in terms of SSL/TLS it’s only about handling HTTP traffic through it. Therefore, let’s go with ‘HTTPS’ in the docs (e.g., ‘HTTPS port’ is fine with me). We do need to reference the term SSL (TLS) as well, as the modules now have `ssl` in the name.

Let’s go with ‘HTTPS’ in the docs, 
‘HTTPS’ makes sense, as Zotonic is only about HTTP over SSL/TLS. Our modules now have `ssl` in the name.

/cc @CyBeRoni Any input on